plugin/entrySchema.go

```diff
@@ -67,7 +75,9 @@ func schema(e Entry) (*EntrySchema, error) {
 	}
 }
 
-type entrySchema struct {
+// EntrySchemaData is the raw data we intend to serialize. You should
```

Comment:
I could update this comment to reflect that we use this as part of constructing nodes in the external plugin graph, and it needs to be kept in-sync with data we store in EntrySchema.